ddl/backfilling.go

```diff
@@ -572,6 +595,131 @@ func setSessCtxLocation(sctx sessionctx.Context, info *reorgInfo) error {
 	return nil
 }
 
+type backfillScheduler struct {
+	ctx          context.Context
+	reorgInfo    *reorgInfo
+	sessPool     *sessionPool
+	tp           backfillWorkerType
+	tbl          table.PhysicalTable
+	decodeColMap map[int64]decoder.Column
+	jobCtx       *JobContext
+
+	workers []*backfillWorker
+	maxSize int
+
+	taskCh   chan *reorgBackfillTask
+	resultCh chan *backfillResult
+}
+
+const backfillTaskChanSize = 1024
+
+func newBackfillScheduler(ctx context.Context, info *reorgInfo, sessPool *sessionPool,
+	tp backfillWorkerType, tbl table.PhysicalTable, decColMap map[int64]decoder.Column,
+	jobCtx *JobContext) *backfillScheduler {
+	return &backfillScheduler{
+		ctx:          ctx,
+		reorgInfo:    info,
+		sessPool:     sessPool,
+		tp:           tp,
+		tbl:          tbl,
+		decodeColMap: decColMap,
+		jobCtx:       jobCtx,
+		workers:      make([]*backfillWorker, 0, 4),
```

Comment:
here the 4 workers is expect number of worker?

Reply:
Changed to `tidb_ddl_reorg_worker_cnt`.